packages/nodejs/test/clientV2.test.ts

```diff
@@ -234,6 +230,100 @@ test.each([clientV2, clientWeb])(
     }
 );
 
+test.each([clientV2, clientWeb])(
+    'getChainParameters corresponds to GetBlockSummary subset',
+    async (client) => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const blockSummary: any = await clientV1.getBlockSummary(testBlockHash);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const chainParameters: any = await client.getBlockChainParameters(
+            testBlockHash
+        );
+
+        const foundationAccount = (
+            await client.getAccountInfo(
+                blockSummary.updates.chainParameters.foundationAccountIndex
+            )
+        ).accountAddress;
+        expect(chainParameters.foundationAccount).toEqual(foundationAccount);
+        blockSummary.updates.chainParameters.foundationAccountIndex = undefined;
+        chainParameters.foundationAccount = undefined;
+
+        expect(blockSummary.updates.chainParameters).toEqual(chainParameters);
+    }
+);
+
+test.each([clientV2, clientWeb])(
+    'getChainParameters corresponds to GetBlockSummary subset on protocol level < 4',
+    async (client) => {
+        const oldBlockHash =
+            'ed2507c4d05108038741e87757ab1c3acdeeb3327027cd2972666807c9c4a20d';
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const blockSummary: any = await clientV1.getBlockSummary(oldBlockHash);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const chainParameters: any = await client.getBlockChainParameters(
+            oldBlockHash
+        );
+
+        const foundationAccount = (
+            await client.getAccountInfo(
+                blockSummary.updates.chainParameters.foundationAccountIndex
+            )
+        ).accountAddress;
+        expect(chainParameters.foundationAccount).toEqual(foundationAccount);
+        blockSummary.updates.chainParameters.foundationAccountIndex = undefined;
+        chainParameters.foundationAccount = undefined;
+
+        expect(blockSummary.updates.chainParameters).toEqual(chainParameters);
+    }
+);
+
+test.each([clientV2, clientWeb])(
+    'getPoolInfo corresponds to getPoolStatus with a bakerId',
+    async (client) => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why this any? And if they return the same object, why not keep the old function name?

Reply:
It's fine